src/internal/polyfills/String.padCharsEnd.js

```diff
@@ -0,0 +1,21 @@
+const padCharsEnd = (padString, targetLength, value) => {
+  // eslint-disable-next-line no-bitwise
+  let finalLength = targetLength >> 0;
+  let finalPadString = String(
+    typeof padString !== 'undefined' ? padString : ' '
+  );
+
+  if (value.length > finalLength) {
+    return String(value);
+  }
+  finalLength -= value.length;
+  if (finalLength > finalPadString.length) {
+    finalPadString += finalPadString.repeat(
```

Comment:
String.repeat may not be available, so we have two options here:

**1.** use always polyfill of [String.prototype.repeat](https://github.com/char0n/ramda-adjunct/blob/f1c5dfaac3b38a5599a6bb71a570205b025c7112/src/internal/polyfills/String.repeat.js#L50-L49) here that we already have in codebase

**2.** use polyfill of [String.prototype.repeat](https://github.com/char0n/ramda-adjunct/blob/f1c5dfaac3b38a5599a6bb71a570205b025c7112/src/internal/polyfills/String.repeat.js#L50-L49) only when `repeat` method on string is not available. 

I'll leave the decission and implementation up to you. IMHO best approach is second option